.github/workflows/coverage.yml

```diff
@@ -9,7 +9,51 @@ on:
   pull_request:
 
 jobs:
-  call_code_coverage:
+  combined_coverage:
     uses: yext/slapshot-reusable-workflows/.github/workflows/coverage.yml@v1
+    with:
+      test_script: npm run test
```

Comment:
I thought this was already the default test script?

Reply:
nope, the default is `npx jest --coverage`